parameters_meta_schema.json

```diff
@@ -0,0 +1,158 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$id": "https://nextflow.io",
+  "title": "Nextflow Schema Meta-schema",
+  "description": "Meta-schema to validate Nextflow parameter schema files",
+  "type": "object",
+  "properties": {
+    "$schema": {
+      "title": "schema",
+      "type": "string",
+      "minLength": 1
+    },
+    "$id": {
+      "title": "ID URI",
+      "type": "string",
+      "minLength": 1
+    },
+    "title": {
+      "title": "Title",
+      "type": "string",
+      "minLength": 1
+    },
+    "description": {
+      "title": "Description",
+      "type": "string",
+      "minLength": 1
+    },
+    "type": {
+      "title": "Top level type",
+      "type": "string",
+      "const": "object"
+    },
+    "definitions": {
+      "title": "Parameter groups",
+      "type": "object",
+      "patternProperties": {
+        "^.*$": {
+          "type": "object",
+          "required": [
+            "title",
+            "type",
+            "properties"
+          ],
+          "properties": {
+            "title": {
+              "type": "string",
+              "minLength": 1
+            },
+            "type": {
+              "const": "object"
+            },
+            "fa_icon": {
+              "type": "string",
+              "pattern": "^fa"
+            },
+            "description": {
+              "type": "string",
+              "minLength": 1
+            },
+            "required": {
+              "type": "array"
+            },
+            "properties": {
+              "type": "object",
+              "patternProperties": {
+                "^.*$": {
+                    "type": "object",
+                    "required": [
+                      "type"
+                    ],
+                    "properties": {
+                      "type": {
+                        "type": "string",
+                        "enum": ["string", "boolean", "integer", "number"]
+                      },
+                      "format": {
+                        "type": "string",
+                        "enum": ["file-path", "directory-path", "path"]
```

Comment:
```suggestion
                        "enum": ["file-path", "directory-path", "path", "file-path-pattern"]
```

Reply:
Yeah I wrote the draft of this doc before that existed! 😅

Reply:
aaah that explains it :wink: